js/ui/payg.js

```diff
@@ -660,7 +660,7 @@ class PaygNotifier extends GObject.Object {
 
     notify(secondsLeft) {
         // Only notify when in an regular session, not in GDM or initial-setup.
-        if (Main.sessionMode.currentMode != 'user') {
+        if (Main.sessionMode.currentMode != 'endless') {
```

Comment:
Why is the session mode `endless`? Is that one of our downstream patches? Maybe this should be a fixup commit if so.

Reply:
This is ref. to this change : https://phabricator.endlessm.com/T29401 - i could add this as a fixup commit, but the change is part of debian-master instead of master

Reply:
Agreed, it wouldn't make sense to make this a fixup; the change that's on master is a patch copied from Ubuntu (https://github.com/endlessm/gnome-shell/pull/627/commits/7bff1f25b5a6ba03125a1af3b6494aeda1c38361). And now I see you already link to T29401 in the commit message. So LGTM but Georges would know better.